### PR TITLE
Created Bio for myearick

### PR DESCRIPTION
**Pull request by myearick**
Just added a short bio

### CODE REVIEW

**Comment by rfriedel** [comment]
Nice Bio!

File: bios/ myearick.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Name:  Mark Yearick
```
Comment (chotai) [inline bios/ myearick.md:1]: Cool bio bro! :100: :+1: :

**Comment by ghost** [comment]
:+1:

**Comment by githubteacher** [comment]
:+1:

**Comment by ChuaMichelle** [comment]
:8ball:

**Comment by dazahern** [comment]
:+1:

**Pull request merged**